src/PhpEnvironment/Response.php

```diff
@@ -74,6 +80,23 @@ public function contentSent()
         return $this->contentSent;
     }
 
+    /**
+     * @param callable $handler
+     */
+    public function setHeadersSentHandler($handler)
```

Comment:
`callable` has been a valid typehint since at least 5.6, if not earlier; we should use it here. That way you don't even need to check the type within this method.

Also, document this as `@return void`, please.

Reply:
implemented.